plugins/basestate/BaseState.cc

```diff
@@ -99,6 +97,11 @@ class BaseState : public System,
         auto model = Model(_entity);
         this->m_baseLink = model.LinkByName(_ecm, baseLinkName);
 
+        // Enable velocity computation in Gazebo
+        Link(this->m_baseLink).EnableAccelerationChecks(_ecm, true);
```

Comment:
```suggestion
        Link(this->m_baseLink).EnableVelocityChecks(_ecm, true);
```

typo?

Reply:
Yep nice catch! Thank you!

Reply:
Solved in 5e0ec19563fa13c163c27bd1abc2926b8b88523c